test/helper/test_timeline.js

```diff
@@ -99,7 +101,8 @@ const testTimelineFiles = (timelineFiles) => {
         let timeline;
 
         before(async () => {
-          const importPath = '../../' + path.relative(process.cwd(), triggersFile).replace(/\\/g, '/');
+          // Normalize path
+          const importPath = '../../' + path.relative(process.cwd(), triggersFile).replace(/\\/g, '/').replace('.ts', '.js');
```

Comment:
Also as I'm looking at this, is this a case for `path.posix.relative` and then not needing the replacing backslashes with slashes?